modules/nodes-sigstore-configure-image-policy.adoc

```diff
@@ -0,0 +1,251 @@
+<9> Optional: Specifies one of the following processes to verify the identity in the signature and the actual image identity:
+* `MatchRepoDigestOrExact`.
+* `MatchRepository`.
+* `ExactRepository`. The `exactRepository` parameter must be specified.
+* `RemapIdentity`. The `prefix` and `signedPrefix` parameters must be specified.
+--
++
+--
+.Example image policy object with a Fulcio certificate policy and the `ExactRepository` match policy
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1alpha1
+kind: ImagePolicy <1>
+metadata:
+  name: p1
+  namespace: mynamespace <2>
+spec:
+  scopes: <3>
+    - example.io/crio/signed
+  policy: <4>
+    rootOfTrust: <5>
+      policyType: FulcioCAWithRekor <6>
+      fulcioCAWithRekor: <7>
+        fulcioCAData: a2V5RGF0YQ==
+        fulcioSubject:
+          oidcIssuer: "https://expected.OIDC.issuer/"
+          signedEmail: "expected-signing-user@example.com"
+        rekorKeyData: cmVrb3JLZXlEYXRh <8>
+    signedIdentity:
+      matchPolicy: ExactRepository <9>
+      exactRepository:
+        repository: quay.io/crio/signed <10>
+----
+<1> Creates an `ImagePolicy` object.
+<2> Specifies the namespace where the image policy is applied.
+<3> Defines a list of repositories or images assigned to this policy.
+<4> Specifies the parameters that define how the images are verified.
+<5> Defines a root of trust for the policy.
+<6> Specifies the policy types that define the root of trust, either a public key or a link:https://docs.sigstore.dev/certificate_authority/overview/[Fulcio certificate]. Here, a Fulcio certificate with required Rekor verification.
+<7> For a Fulcio certificate policy, the following parameters are required:
+* `fulcioCAData`: Specifies a base64-encoded Fulcio certificate in the PEM format. The maximum length is 8192 characters.
+* `fulcioSubject`: Specifies the OIDC issuer and the email of the Fulcio authentication configuration.
+<8> Specifies a base64-encoded Rekor public key in the PEM format. This parameter is required when when the `policyType` is `FulcioCAWithRekor`. The maximum length is 8192 characters.
+<9> Optional: Specifies one of the following processes to verify the identity in the signature and the actual image identity:
+* `MatchRepoDigestOrExact`.
+* `MatchRepository`.
+* `ExactRepository`. The `exactRepository` parameter must be specified.
+* `RemapIdentity`. The `prefix` and `signedPrefix` parameters must be specified.
+<10> For the `exactRepository` match policy, specifies the repository that contains the image identity and signature.
+--
+
+. Create the image policy object:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+The Machine Config Operator (MCO) updates the machine config pools (MCP) in your cluster.
+
+.Verification
+
+. After the nodes in your cluster are updated, you can verify that the image policy has been configured:
```

Comment:
Since this is a single step (with multiple sub-steps), IMO, a bullet can be used here.